### PR TITLE
CORE-2381 - add compliance rule pack

### PR DESCRIPTION
**Pull request by charlieduong94**
Adds a rule pack that users can import to get alerts on open compliance gaps.

Keeping it simple for now, we can add additional alerts later.

### CODE REVIEW

**Comment by charlieduong94** [comment]
Will version after approval

**Review by erkangz — APPROVED** [review]

File: rule-packs/compliance.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+    "name": "compliance-gaps",
+    "description": "Finds open compliance gaps.",
+    "queries": [
+      {
+        "name": "gaps",
+        "query": "FIND jupiterone_compliance_gap",
+        "version": "v1"
+      }
+    ],
+    "alertLevel": "HIGH"
```
Comment (erkangz) [inline rule-packs/compliance.json:12]: Actually perhaps this is better as Medium?

Comment (charlieduong94) [reply rule-packs/compliance.json:12]: I'm cool with that

Comment (charlieduong94) [reply rule-packs/compliance.json]: done

**Review by erkangz — APPROVED** [review]

**Pull request merged**